<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Countdown</class>
 <widget class="QWidget" name="Countdown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>278</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/resources/icons/colorful/logo.svg</normaloff>:/resources/icons/colorful/logo.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="window_bg" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="hl_description">
        <item>
         <widget class="QLabel" name="lb_text1">
          <property name="text">
           <string>距离</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_event">
          <property name="text">
           <string>{}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_text2">
          <property name="text">
           <string>还有</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lb_targetddate">
          <property name="text">
           <string>{}</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lb_CountDown">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>{}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="action_open_config">
   <property name="text">
    <string>配置</string>
   </property>
   <property name="toolTip">
    <string>配置这个计时器</string>
   </property>
  </action>
  <action name="action_save_profile">
   <property name="text">
    <string>保存当前窗口配置</string>
   </property>
   <property name="toolTip">
    <string>将当前窗口的大小、位置保存到配置文件中</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
